spec/models/veteran_onboarding_spec.rb

```diff
@@ -20,10 +22,43 @@
     end
   end
 
+  it 'creates a VeteranOnboarding object if toggle is enabled' do
+    Flipper.enable(:veteran_onboarding_beta_flow, user)
+    expect { VeteranOnboarding.for_user(user) }.to change(VeteranOnboarding, :count).by(1)
+  end
+
   describe '#show_onboarding_flow_on_login' do
     it 'returns the value of display_onboarding_flow' do
-      subject = described_class.new(display_onboarding_flow: true, user_account:)
+      subject = described_class.for_user(user)
+      Flipper.enable(:veteran_onboarding_beta_flow, user)
       expect(subject.show_onboarding_flow_on_login).to be_truthy
```

Comment:
I see you're enabling `veteran_onboarding_beta_flow` here, but somehow when I run the test, the corresponding code in veteran_onboarding.rb is not touched 😕 
![image](https://github.com/department-of-veterans-affairs/vets-api/assets/10993987/a95c075e-dcd3-4644-b5c2-26f0933e4a1b)

Can you try to get the spec to test that `elsif` block as well as the `else` block?

Reply:
I forget that adding a feature toggle in always has it being set to `true` by default.  I went ahead and disabled the new flag in the `user_spec.rb` and those line should now be covered

Reply:
awesome 👍 I see it's covered. 

Can you add one for the `else`? This is what I see when I run `spec/models` (ignore the binding.prys 😅):
![image](https://github.com/department-of-veterans-affairs/vets-api/assets/10993987/c96e76b7-64c0-49c0-b79c-2ed108a45b31)

Reply:
Explanation: when we retrieve the onboarding object [for the user](https://github.com/department-of-veterans-affairs/vets-api/blob/VA-IIR-712_newly_onboarded_veteran/app/models/veteran_onboarding.rb#L46), we return `nil` if the flags are disabled.

Reply:
ah, okay! thanks for the explanation!!